data/js/controls.js

```diff
@@ -639,27 +650,29 @@ function buttonclick(number, isdown) {
 }
 
 function padclick(type, number, isdown) {
-  switch (type) {
-    case CENTER:
-      if (isdown) websock.send("pcdown:" + number);
-      else websock.send("pcup:" + number);
-      break;
-    case UP:
-      if (isdown) websock.send("pfdown:" + number);
-      else websock.send("pfup:" + number);
-      break;
-    case DOWN:
-      if (isdown) websock.send("pbdown:" + number);
-      else websock.send("pbup:" + number);
-      break;
-    case LEFT:
-      if (isdown) websock.send("pldown:" + number);
-      else websock.send("plup:" + number);
-      break;
-    case RIGHT:
-      if (isdown) websock.send("prdown:" + number);
-      else websock.send("prup:" + number);
-      break;
+  if(!$("#id" + number + " nav").hasClass("disabled")) {
```

Comment:
you could use an early return here :-)

Reply:
I'll fold this change into my next pr ;)